### PR TITLE
tmuxPlugins.vim-tmux-focus-events: init at unstable 2010-10-06

### PR DESCRIPTION
**Pull request by RonanMacF**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/tmux-plugins/default.nix [hunk 1]
```diff
@@ -392,6 +392,24 @@ in rec {
...
+      sha256 = "130l73v18md95djkc4s9d0fr018f8f183sjcgy7dgldwdaxlqdi1";
+    };
+    meta = {
+      homepage = "https://github.com/tmux-plugins/vim-tmux-focus-events";
+      description = "Makes FocusGained and FocusLost autocommand events work in vim when using tmux";
+      license = stdenv.lib.licenses.mit;
+      platforms = stdenv.lib.platforms.unix;
+      maintainers = with stdenv.lib.maintainers; [ ronanmacf ];
+    };
+  };
```
Comment (SuperSandro2000) [inline pkgs/misc/tmux-plugins/default.nix:411]:
```suggestion
  vim-tmux-focus-events = mkDerivation {
    pluginName = "vim-tmux-focus-events";
    version = "unstable-2020-10-05";

    src = fetchFromGitHub {
      owner = "tmux-plugins";
      repo = "vim-tmux-focus-events";
      rev = "a568192ca0de4ca0bd7b3cd0249aad491625c941";
      sha256 = "130l73v18md95djkc4s9d0fr018f8f183sjcgy7dgldwdaxlqdi1";
    };

    meta = with stdenv.lib; {
      homepage = "https://github.com/tmux-plugins/vim-tmux-focus-events";
      description = "Makes FocusGained and FocusLost autocommand events work in vim when using tmux";
      license = licenses.mit;
      platforms = platforms.unix;
      maintainers = with maintainers; [ ronanmacf ];
    };
  };
```

Comment (RonanMacF) [reply pkgs/misc/tmux-plugins/default.nix]: whoops I thought that would merge into the original commit. Is it ok as is or do I need to squash the commit?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Can you please squash the commits together?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102661` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.vim-tmux-focus-events</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102661` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.vim-tmux-focus-events</li>
  </ul>
</details>

**Comment by RonanMacF** [comment]
> Can you please squash the commits together?

done

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102661` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.vim-tmux-focus-events</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 102661` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tmuxPlugins.vim-tmux-focus-events</li>
  </ul>
</details>

**Pull request merged**